operators/multiclusterobservability/manifests/base/alertmanager/alert_rules.yaml

```diff
@@ -48,3 +48,33 @@ data:
           labels:
             slo: "API Server Request Duration"
             target: 0.9900
+      - name: grafana-dashboard
```

Comment:
is it possible to use recording rule in managed cluster side? do some aggregation in managed cluster to create new cluster-level metrics, then in hub side, when we try to query among all clusters, the query will be lightweight.
I have this concern because if those queries are heavy, to add them as recording rule in thanos rule means those queries will be performed every 30 seconds, that will lead to much resource consumption.
@morvencao @clyang82 comments?

Reply:
the reason why I added these to the thanos side insteafd of managed cluster side is that the queries need do aggregation/filter with the cluster label. I didn't test the overhead to the hub side for these added recordingrules.